packages/carbon-web-components/src/components/structured-list/structured-list.ts

```diff
@@ -52,11 +64,32 @@ class BXStructuredList extends FocusMixin(LitElement) {
     return true;
   }
 
+  updated(changedProperties) {
+    const attributes = ['condensed', 'flush'];
+    attributes.forEach((attr) => {
+      if (changedProperties.has(attr)) {
+        // Propagate watched attribute to descendants until `:host-context()` gets supported in all major browsers
+        forEach(
+          this.querySelectorAll(
+            (this.constructor as typeof BXStructuredList).selectorRowsWithHeader
+          ),
+          (elem) => {
```

Comment:
Wondering if it might be better to do this in `firstUpdated` instead, or save all the `rowsWithHeader` components to variable/property so we don't have to re-query them every time the component is updated.

Reply:
Hm, I can see how that would affect it. We can look into that closer in some future release since it's not too performance heavy.